app/common/components/kendo-grid/editable-cell/get-action-cell.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+
+import { GridCell, GridCellProps } from '@progress/kendo-react-grid';
+
+export type ActionsComponent = React.ComponentType<GridCellProps>;
+
+interface GetActionCellParams {
+    actions?: ActionsComponent;
+}
+
+export function getActionCell({ actions: Actions }: GetActionCellParams) {
```

Comment:
This component doesn't look useful. Wouldn't it be better to use a closure with handlers here? 
```
function getActionCell(onEdit, onSave, onCancel, onDelete) {
    return class extends GridCell {
        // ...
    };
}
```